README.md

```diff
@@ -58,11 +38,50 @@ To build for production, just run the `build` script:
 $ yarn build
 ```
 
-This will create a `dist` folder with everything, including compressed
-versions of all files that are at least 1024 bytes in size.  There are
-two compressed files per original file: a zopfli (.gz) file and a
-brotli (.br) file.  If a file doesn't compress well, there won't be a
-compressed version of it.
+You will need to configure base URL and Firebase config (see Configuration
+below).
+
+This will create a `dist` folder with everything, including compressed versions
+of all files that are at least 1024 bytes in size.  There are two compressed
+files per original file: a zopfli (.gz) file and a brotli (.br) file.  If a file
+doesn't compress well, there won't be a compressed version of it.
+
+## Configuration
+
+There are two ways to configure the environment.  For basic configuration, you
+can use environment variables:
+
+- `WEBSHOP_BASE_URL`: The base URL to the backend.
+- `WEBSHOP_FIREBASE_CONFIG`: The Firebase config.  The config needs to be valid
+  JSON, so remember when copying the config from the Firebase console to change
+  the keys to be double-quoted strings (this is only necessary when using an
+  environment variable).
```

Comment:
Legg med eksempel dotenv med alle nøkler som kan settes. Uten å ha satt verdier da. Vanlig praksis for å vise konfigurasjonsmuligheter

Reply:
Er dette egentlig nødvendig når alle nøklene er listet opp her?

Reply:
Ikke nødvendig, men nice to have i oppstart for å kopiere og rename. Ikke veldig nøye for meg

Reply:
Jeg er bare usikker på hva det har for seg siden verdiene ikke bør være der. Det anbefalte oppsettet er med en lokal webpack-config under utvikling, siden den er mye mer fleksibel.

Reply:
Jeg foretrekker som regel å prøve å gjøre det likt som når man ruller ut. Om det er stor forskjell på hvordan det konfigureres opp og fragmenterer implementasjon der er det en unødvendig kilde til feil. Men som sagt, ikke en bakketopp jeg er villig til å bli gammel på